controller/articles.controller.js

```diff
@@ -24,8 +26,12 @@ exports.getArticles = (req, res, next) => {
 
 exports.getCommentsByArticleId = (req, res, next) => {
   const { article_id } = req.params;
+
   fetchCommentsByArticleId(article_id)
     .then((comments) => {
+      if (comments.length === 0) {
```

Comment:
I don't think this code block is necessary because you'd just send back whatever comments is regardless